code/controllers/subsystems/job.dm

```diff
@@ -159,31 +159,14 @@ SUBSYSTEM_DEF(jobs)
 			if(!candidates.len)
 				continue
 
-			// Build a weighted list, weight by age.
+			// Build a list of candidates.
 			var/list/weightedCandidates = list()
 			for(var/mob/V in candidates)
-				// Log-out during round-start? What a bad boy, no head position for you!
+				// Log-out during round-start? No point in considering the player for candidacy.
 				if(!V.client)
 					continue
 
-				var/age = V.client.prefs.age
-
-				var/min_job_age = job.get_minimum_character_age(V.get_species())
-				var/ideal_job_age = job.get_ideal_character_age(V.get_species())
-
-				if(age > (ideal_job_age + 20)) // Elderly for the position
-					weightedCandidates[V] = 3
-				else if(age > (ideal_job_age + 10)) // Good, but on the elderly side
-					weightedCandidates[V] = 6
-				else if(age > (ideal_job_age - 10)) // Perfect
-					weightedCandidates[V] = 10
-				else if(age > (min_job_age + 10)) // Good, but on the young side
-					weightedCandidates[V] = 6
-				else if(age >= min_job_age) // Too young
-					weightedCandidates[V] = 3
-				else
-					if(candidates.len == 1) // There's only one option
-						weightedCandidates[V] = 1
+				weightedCandidates[V] = 1
```

Comment:
not the cleanest solution. maybe i could `pick()` straight from the `candidates` list?